site/app/libraries/FileUtils.php

```diff
@@ -566,4 +566,77 @@ public static function areWordsInFile(string $file, array $words)
 
         return $words_detected;
     }
+
+    /**
+     * Given an array of uploaded files, makes sure they are properlly uploaded
+     *
+     * @param array $files - should be in the same format as the $_FILES[] variable
+     * @return array representing the status of each file
+     * e.g. array('name' => 'foo.txt','type' => 'application/octet-stream', 'error' =>      
+     *            'success','size' => 100, 'success' => true)
+     * if $files is null returns failed => no files sent to validate
+     */
+    public static function validateUploadedFiles($files){
+
+        if(!isset($files)){
```

Comment:
Let's change this to be `!empty($files)` as the message applies to the empty `[]` as well.